src/main/python/gemini/debugging.py

```diff
@@ -0,0 +1,29 @@
+import helper
```

Comment:
Should there be a comment to explain this file's usage? Otherwise it doesn't seem as useful to add to source maybe?